Ui/DataProvider/Modifier/Notifications.php

```diff
@@ -173,18 +173,5 @@ private function hideNotification(array $meta)
     private function getNotificationContent()
     {
         return '';
-        $cacheKey = 'buckaroo-release-notification';
```

Comment:
are we using this functionality?

Reply:
no, as you can see it was set return before this statement, but t was used more than 1 year ago
now it's not needed